src/tests/ftest/pool/global_handle.py

```diff
@@ -25,38 +25,31 @@
 
 import os
 import traceback
-import json
 
 from apricot import TestWithServers
 
 import check_for_pool
-from daos_api import DaosContext, DaosPool, DaosContainer, DaosApiError
+from daos_api import DaosPool, DaosContainer, DaosApiError
 
-def check_handle(buf_len, iov_len, buf, uuidstr, rank):
+def check_handle(self, buf_len, iov_len, buf, uuidstr, rank):
```

Comment:
This function needs to move an become a method of the `GlobalHandle` class

Reply:
This function still needs to be defined inside of the `GlobalHandle` class, as it is currently defined outside of the class.

Also, to make use of the existing `TestWithServers.tearDown()` functionality the local `pool` variable should be `self.pool`.